### PR TITLE
HW-152919: Add Confirmation Popup

### PR DESCRIPTION
**Pull request by vmhenryli**
1. Add confirmation_popup in cardAction, so that end users have opportunities to cancel or confirm operations they chose.

2. Defined confirmationPopup with title, text, confirmation_button_text, and cancel_button_text.

### CODE REVIEW

**Comment by vmhenryli** [comment]
@jmaite @bharath2104 Could you review this modification? Thank you.

**Comment by jmaite** [comment]
@rworsnop can you please merge this?  I don't have access to do so.

**Review by bharath2104 — APPROVED** [review]

**Review by shsmysore — APPROVED** [review]

**Review by jonathanrfisher1 — APPROVED** [review]
Makes sense to me!  I tried to think of anything else you might want in a confirmation dialog and could only think of colors for buttons etc, but I suspect that would be handled in its own way really.

**Comment by vmhenryli** [comment]
@bharath2104 @shsmysore @jonathanrfisher1 Thank you!

Could you please merge this pull request if you have write access to this repository?

**Comment by jonathanrfisher1** [comment]
> @bharath2104 @shsmysore @jonathanrfisher1 Thank you!
> 
> Could you please merge this pull request if you have write access to this repository?

Looks like you need Rob for the that.

**Pull request merged**
